src/options/modules/ManualAdjustments.js

```diff
@@ -0,0 +1,18 @@
+import { getBrowserValue } from "/common/modules/BrowserCompat.js";
+
+/**
+ * Initializes module.
+ *
+ * Applies the adjustments.
+ *
+ * @returns {Promise}
+ */
+export function init() {
+    getBrowserValue({
+        firefox: "https://addons.mozilla.org/firefox/addon/unicodify-text-transformer/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-addon-settings-inline&utm_campaign=unicodify-addon-settings-inline",
+        thunderbird: "https://addons.thunderbird.net/thunderbird/addon/unicodify-text-transformer/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-addon-settings-inline&utm_campaign=unicodify-addon-settings-inline",
+        chrome: "https://chrome.google.com/webstore/detail/unicodify-text-transformer/",
+    }).then((browserUrl) => {
+        document.getElementById("link-unicodify").href = browserUrl;
```

Comment:
Hmm I don't see that ID being used anywhere in the options.html?

Reply:
Oops, this ID would be used by #93, so I will comment this line out and add a comment. Alternatively, I could add something like this to `options.html`:
```html
<span class="helper-text">For Unicode font conversion from the context menu, please try our <strong>υ</strong> <a id="link-unicodify">Unicodify</a> add-on.</span>
```